specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-04-10-privatepreview/GeoRestore.json

```diff
@@ -0,0 +1,241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PostgreSQLServerManagementClient",
+    "description": "The Microsoft Azure management API provides create, read, update, and delete functionality for Azure PostgreSQL resources including servers, databases, firewall rules, VNET rules, security alert policies, log files and configurations with new business model.",
+    "version": "2021-04-10-privatepreview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBForPostgreSql/flexibleServers/{serverName}": {
+      "put": {
+        "tags": [
+          "GeoRestore"
+        ],
+        "operationId": "Servers_GeoCreate",
+        "x-ms-examples": {
+          "Create a database as a Geo restore": {
+            "$ref": "./examples/ServerCreateGeoRestore.json"
+          }
+        },
+        "description": "Creates a new server.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../2020-02-14-privatepreview/postgresql.json#/parameters/ServerNameParameter"
```

Comment:
We are referencing something from `2020-02-14-privatepreview/postgresql.json` here but actually the tag is using `2020-02-14-preview/postgresql.json`, and this might be the root cause of the LintDiff errors and Track 2 Validation errors